perovskite_solar_cell_database/schema_sections/ions/ion.py

```diff
@@ -0,0 +1,166 @@
+from nomad.datamodel.metainfo.basesections import PureSubstanceSection
+from nomad.metainfo import Quantity
+import openpyxl
+import os
+from rdkit import Chem
+from rdkit.Chem import AllChem
+from ase import Atoms
+
+
+class Ion(PureSubstanceSection):
+    """
+    A section describing the ions used in the solar cell.
+    """
+    ion_type = Quantity(
+        type=str,
+        shape=[],
+        description='Type of the ion.',
+        a_eln=dict(
+            component='StringEditQuantity')
+    )
+
+    common_name = Quantity(
+        type=str,
+        description='Common name.',
+        a_eln=dict(
+            component='StringEditQuantity'),
+    )
```

Comment:
At some point, I would recommend migrating to the [new plugin template](https://github.com/nomad-coe/nomad-schema-plugin-example) that incorporates also e.g. automated styling via `ruff`. This will basically enforce the same style within all of our code. Does not have to be in this PR.